### ISSUE TITLE
Does it properly handle branches in multiple repos?

### ISSUE THREAD

**Issue by andirapandi** [hraban/tomono#34]
Dear all,

reading the readme, this tool supports multiple repos with multiple branches each, merging all of them into a new repo with even more branches.

Is that correct?

I am asking since that seems to be really hard - could not find that with many other options/tools here and also googling for quite a while.

Also, are the branches fully functioning afterwards or are there any restrictions? One of the issues here says that branches cannot be merged anymore post migration.

Many Thanks!
Andreas

**Comment by hraban** [comment]
yes, the blog post linked in the readme has the details. it merges everything in the final mono repository, history is preserved, commit hashes are preserved, etc, again see the readme. branches can be merged, as explained in the readme section.

I strongly recommend trying it yourself, and not taking my word for it. ;)

**Comment by andirapandi** [comment]
It does work. Even in GIT bash in Windows (which comes with standard GIT installation).
For Folder names containing spaces, I had to escape them using \  backslash.
First look: very good. Now testing the repo.
Merged the result of git-tfs.
BTW, this is never given as a solution for multi branch repo merging - at least I only stumbled across it after searching far and wide.
TFS supports multiple branches on multiple levels, which causes the issues when converting to GIT.

Many thanks!
